test/Azure.Functions.Cli.Tests/E2E/StartTests.cs

```diff
@@ -338,42 +338,172 @@ await CliTester.Run(new RunConfiguration
-                        result.Should().Be("Hello, Test. This HTTP triggered function executed successfully.", because: "response from default function should be 'Hello, {name}. This HTTP triggered function executed successfully.'");
+                        result.Should().Be("Welcome to Azure Functions!", because: "response from default function should be 'Welcome to Azure Functions!'");
 
                         if (_output is Xunit.Sdk.TestOutputHelper testOutputHelper)
                         {
-                            testOutputHelper.Output.Should().Contain($"{Constants.FunctionsInProcNet8Enabled} app setting enabled in local.settings.json");
-                            testOutputHelper.Output.Should().Contain("Starting child process for in-process model host");
-                            testOutputHelper.Output.Should().Contain("Started child process with ID");
+                            testOutputHelper.Output.Should().Contain("4.10");
+                            testOutputHelper.Output.Should().Contain("Selected out-of-process host");
                         }
                     }
                 },
-                CommandTimeout = TimeSpan.FromSeconds(300),
+                CommandTimeout = TimeSpan.FromSeconds(900),
+            }, _output);
+        }
+
+        [Fact]
+        public async Task start_dotnet_isolated_csharp_with_oop_host_without_runtime_specified()
+        {
+            await CliTester.Run(new RunConfiguration
+            {
+                Commands = new[]
+                {
+                    "init . --worker-runtime dotnet-isolated",
+                    "new --template Httptrigger --name HttpTrigger",
+                    "start --port 7073 --verbose"
+                },
+                ExpectExit = false,
+                Test = async (workingDir, p) =>
+                {
+                    using (var client = new HttpClient() { BaseAddress = new Uri("http://localhost:7073") })
+                    {
+                        (await WaitUntilReady(client)).Should().BeTrue(because: _serverNotReady);
+                        var response = await client.GetAsync("/api/HttpTrigger?name=Test");
+                        var result = await response.Content.ReadAsStringAsync();
+                        p.Kill();
+                        await Task.Delay(TimeSpan.FromSeconds(2));
+                        result.Should().Be("Welcome to Azure Functions!", because: "response from default function should be 'Welcome to Azure Functions!'");
+
+                        if (_output is Xunit.Sdk.TestOutputHelper testOutputHelper)
+                        {
+                            testOutputHelper.Output.Should().Contain("4.10");
+                            testOutputHelper.Output.Should().Contain("Selected out-of-process host");
+                        }
+                    }
+                },
+                CommandTimeout = TimeSpan.FromSeconds(900),
```

Comment:
These are some high timeouts on all the tests. Is this required? If possible, let's avoid this, but this is certainly something we want to track and fix.